backend/src/main/java/com/redhat/cloud/notifications/models/BehaviorGroup.java

```diff
@@ -91,6 +93,13 @@
     @JsonIgnore
     private Set<EventTypeBehavior> behaviors;
 
+    @JsonInclude
+    @JsonProperty(access = WRITE_ONLY)
```

Comment:
After reading the docs, yes, it should be. 

> (note that the direction of read and write is from perspective of the property, not from external data format: this may be confusing in some contexts).